### ISSUE TITLE
Quickstart example fails on `brig run`

### ISSUE THREAD

**Issue by shimmerjs** [Azure/brigade#254]
Working through the Quickstart docs, everything works fine up until `brig run -f brigade.js deis/empty-testbed`.

- Brigade server is successfully installed from helm chart
- Deployed `brigade-project` out of the `charts/` directory with default `values.yaml`
- copy/pasted `brigade.js` contents from Quickstart guide
- when running `brig run -f brigade.js`, 

```
brig --verbose run -f ./brigade.js deis/empty-testbed
panic: runtime error: slice bounds out of range

goroutine 1 [running]:
github.com/Azure/brigade/brig/cmd/brig/commands.(*scriptRunner).send(0xc4202fd140, 0x7ffeefbff683, 0x12, 0xc420158a00, 0x7a, 0x27a, 0x0, 0x1ce36a2)
	/Users/alex/go/src/github.com/Azure/brigade/brig/cmd/brig/commands/run.go:132 +0x446
github.com/Azure/brigade/brig/cmd/brig/commands.glob..func5(0x23fc700, 0xc4203dbc80, 0x1, 0x4, 0x0, 0x0)
	/Users/alex/go/src/github.com/Azure/brigade/brig/cmd/brig/commands/run.go:84 +0xe5
github.com/Azure/brigade/vendor/github.com/spf13/cobra.(*Command).execute(0x23fc700, 0xc4203dbb80, 0x4, 0x4, 0x23fc700, 0xc4203dbb80)
	/Users/alex/go/src/github.com/Azure/brigade/vendor/github.com/spf13/cobra/command.go:599 +0x3e9
github.com/Azure/brigade/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x23fb820, 0xc420072058, 0x0, 0x1)
	/Users/alex/go/src/github.com/Azure/brigade/vendor/github.com/spf13/cobra/command.go:689 +0x339
github.com/Azure/brigade/vendor/github.com/spf13/cobra.(*Command).Execute(0x23fb820, 0x23fb820, 0xc4204dff50)
	/Users/alex/go/src/github.com/Azure/brigade/vendor/github.com/spf13/cobra/command.go:648 +0x2b
main.main()
	/Users/alex/go/src/github.com/Azure/brigade/brig/cmd/brig/main.go:11 +0x31
```

**Comment by mumoshu** [comment]
@shimmerjs Hi, I've experienced the same issue and solved by building a `brig` command from the git commit tagged v0.7.0, not current master.

Not sure if this is an expected behavior though. 

Hi @technosophos Perhaps brig or api had a backward-incomatible change since then?
I don't really mind about breaking changes myself but let me notice you anyway.

**Comment by technosophos** [comment]
I will test this right now.

**Comment by technosophos** [comment]
Thanks for the quick report. That was my fault do to a recent change. I opened a fix for it and we should have it resolved before we cut 0.8 today.
